debian/control

```diff
@@ -0,0 +1,30 @@
+Source: cri-o
+Section: devel
+Priority: optional
+Maintainer: Lokesh Mandvekar <lsm5@fedoraproject.org>
+Build-Depends: debhelper (>=9),
+    rsync,
+    libdevmapper-dev,
+    libgpgme11-dev,
+    libseccomp-dev,
+    golang,
+    golang-1.8,
```

Comment:
Do you need both `golang` and the versioned `golang-1.8`?  I'd expect `golang` would be sufficient.